app/app.ts

```diff
@@ -151,6 +150,12 @@ function setupIpc(localSettings: LocalSettings, scrSettings: ScrSettings) {
     })
 
   localSettings.on('change', settings => {
+    const loginSettings = {
+      openAtLogin: settings.openAppAtSystemStart,
```

Comment:
Nit: I would rather set this directly inside the `if` condition. It's very likely that if we add more settings to set at login, they would have their own, separate `if` conditions and we couldn't use this general object at all.